Dockerfile

```diff
@@ -44,12 +43,16 @@ COPY templates /var/www/templates
 COPY config /var/www/config
 COPY src /var/www/src
 COPY data /var/www/data
+COPY public /var/www/public
+
+# Copy assets from docker assets stage in a not mapped folder to be maintained
+RUN mkdir -p /assets && mkdir -p /assets/css
+COPY --from=assets /assets/css/*.css /assets/css/
```

Comment:
I copied the assets in earlier, as they tend to change less frequently.

Additionally, I do not understand why you're not moving them to the final, production location, instead of an intermediary location that then requires copying them via the entrypoint script.

Can you elaborate on the rationale a bit more, please?

Reply:
When I use docker-compose.yml, if I don't put the files in an intermediate folder they are lost with the creation of the volume. This means that if I leave as it was previously, in the host folder, the generated assets were not found.

The alternative would be to run `gulp saas` via the entrypoint like `composer install`.

OS: Mac OS
Docker version 18.09.2
docker-compose version 1.23.2